pom.xml

```diff
@@ -333,6 +335,34 @@
     </distributionManagement>
 
     <profiles>
+        <profile>
+            <id>coverage</id>
+            <activation>
```

Comment:
why not use:
```xml
            <activation>
                <property>
                    <name>env.CI</name>
                    <value>true</value>
                </property>
            </activation>
```
like for the `release` profile?

Reply:
If you think this works better, please free free to update the POM. This PR is what we got working on Friday based on some examples from Sonar.

Reply:
@fbiville I do not claim to be a POM expert at all.

Reply:
what this would achieve is enabling this Maven profile by default is the environment variable `CI` has the value `true` (which is the case in GitHub Actions environments), while leaving the profile disabled otherwise (since running code coverage all the time by default seems a bit costly)
let me update the PR
thanks for the fix btw!!